src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -407,15 +416,18 @@ private IDataView GetShuffledData(string path)
 
                         labelBatch[batchIndex] = label;
                         batchIndex += 1;
+                        trainstate.CurrentBatchIndex = batchIndex;
                         // Train.
                         if (batchIndex == batchSize)
                         {
+
+                            learningRate = options.LearningRateScheduler.GetLearningRate(trainstate);
```

Comment:
>learningRate = options.LearningRateScheduler.GetLearningRate(trainstate); [](start = 28, length = 73)

please only do this if scheduling is being used otherwise use constant learning as it was being used before. Passing LR in each run is just inefficient and does not make sense if LR is going to be constant.